### DISCUSSION TITLE
Disabling pre-built UPNP directories

### DISCUSSION THREAD

**Discussion by asazanoff** [gerbera/gerbera#3149] [Q&A]
Hi!

I just start using Gerbera as more powerful replacement for MiniDLNA. 

In MiniDLNA I have an option to set home directory as Video, so I can not see unused Audio and Pictures folders, it is OK! I see there is another discussion https://github.com/gerbera/gerbera/discussions/2837 that covers  this.

Also I have another question. We do not use sorting by date or year, and without previously meant option I have to go to Video/Directories. It is little annoying. 

Is there any options to remove unused directories?

**Comment by KarlStraussberger** [comment]
In fact, the fastest way to achieve this, is to run in `js` `virtual-layout` mode and create your own version of the `importVideo` `import-function`. This gives you full control of the layout. The unused folders are only created for `builtin` layout.

See https://docs.gerbera.io/en/latest/config-import.html and  https://docs.gerbera.io/en/latest/scripting.html for all the options at hand.
